package art.lazying.lazyoracle;

import com.getcapacitor.BridgeActivity;

public class MainActivity extends BridgeActivity {
    @Override public void onCreate(android.os.Bundle state) {
        registerPlugin(NativeArchivePlugin.class);
        super.onCreate(state);
    }
}
